applications/ecallmgr/src/ecallmgr_fs_nodes.erl

```diff
@@ -760,25 +760,47 @@ start_node_stats(#node{}) ->
 -spec start_preconfigured_servers() -> 'ok'.
 start_preconfigured_servers() ->
     kz_util:put_callid(?LOG_SYSTEM_ID),
-    case ecallmgr_config:get(<<"fs_nodes">>) of
-        [] ->
-            lager:info("no preconfigured servers available. Is the sysconf whapp running?"),
-            timer:sleep(5 * ?MILLISECONDS_IN_SECOND),
-            start_preconfigured_servers();
-        Nodes when is_list(Nodes) ->
-            lager:info("successfully retrieved FreeSWITCH nodes to connect with, doing so..."),
-            _ = [kz_util:spawn(fun start_node_from_config/1, [N]) || N <- Nodes],
-            'ok';
-        'undefined' ->
-            lager:debug("failed to receive a response for node configs"),
-            timer:sleep(5 * ?MILLISECONDS_IN_SECOND),
-            start_preconfigured_servers();
-        _E ->
-            lager:debug("received a non-list for fs_nodes: ~p", [_E]),
+    Result = case ecallmgr_config:get(<<"fs_nodes">>) of
+                 [] ->
+                     lager:info("no preconfigured servers available. Is the sysconf whapp running?");
+                 Nodes when is_list(Nodes) ->
+                     lager:info("successfully retrieved FreeSWITCH nodes to connect with, doing so..."),
+                     _ = [kz_util:spawn(fun start_node_from_config/1, [N]) || N <- Nodes],
+                     'success';
+                 'undefined' ->
+                     lager:debug("failed to receive a response for node configs");
+                 _E ->
+                     lager:debug("received a non-list for fs_nodes: ~p", [_E])
+             end,
+    case Result of
+        'success' -> 'ok';
+        _ ->
+            _ = maybe_add_default_fs(),
```

Comment:
@k-anderson what do you mean? the logic here is that first we try a configuration parameter and if it fails for any reason then we try to connect to default node.